code/modules/cargo/packs.dm

```diff
@@ -2734,3 +2734,10 @@ datum/supply_pack/medical/bruisekits
 					/obj/item/toner,
 					/obj/item/toner)
 	crate_name = "toner crate"
+	
+/datum/supply_pack/security/armory/cling_test
+	name = "Changeling Testing Kit"
+	desc = "Contains a single bottle of shiftium, a powerful stimulant with adverse effects on changelings. Because it can turn them in dangerous, horrific monsters, it is often used to expose the identity of changelings. Requires Armory access to open."
+	cost = 20000
```

Comment:
wew, pretty pricey

Reply:
Might as well, given that now you get 2, 3 uses tops out of a bottle, down from 30.

Reply:
Pricy for only 3 uses. I'd say lower it to like 12000-15000